html2text/__init__.py

```diff
@@ -435,6 +435,12 @@ def no_preceding_space(self):
                     self.abbr_title = None
                 self.abbr_data = ''
 
+        def link_url(self, link, title=""):
+            url = urlparse.urljoin(self.baseurl, link)
+            title = ' "{}"'.format(title) if title.strip() else ''
```

Comment:
You can explicitly number the format fields, then it would work for Python 2.6 as well and it's prettier to have them numbered in my opinion.

I'm all for dropping legacy versions of Python, but dropping 2.6 over string format numbering might cause cancer and early death 😄 

However, if that didn't fix it and we see it will get us in more issues, then yeah lets drop Python 2.6.